src/com/itacademy/eshop/product/Toys.java

```diff
@@ -0,0 +1,42 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public class Toys extends Product {
+
+    private int agelimit;
```

Comment:
naming should be in camelCase, every word after the first one should start with a capital letter